src/main/java/frc/robot/RobotContainer.java

```diff
@@ -142,7 +142,11 @@ public RobotContainer() {
     }
     trapElvCommandFactory = new TrapElvCommandFactory(trapElvSubsystem);
     if (enabledSubsystems.turretEnabled) {
-      turretSubsystem = new TurretSubsystem(robotStateManager, null);
+      if (visionSubsystem != null) {
```

Comment:
The visionSubsystem disable got changed such that it is never null, but it could be a default interface class. Double check this will work the way you expect, or potentially change VisionSubsystem back to an abstract class from an interface.